packages/base/big-integer.gts

```diff
@@ -14,7 +14,7 @@ import { TextInputFilter, DeserializedResult } from './text-input-filter';
 function _deserialize(
   bigintString: string | null | undefined
 ): DeserializedResult<bigint> {
-  if (!bigintString) {
```

Comment:
What is wrong with this condition? Doesn't `!bigintString` cover `bigintString == null || bigintString == undefined`?

Reply:
Good question. In fact, this is my pain point. The main issue is of the instance of the NumberCard. At runtime, the number value of 0 is being passed into this function. It turns out 0 is falsy, so when I save 0 in the input text box, the JSON file will show null (which is not expected). 

The types in these _deserialize functions are not reflected correctly. I guess passing number makes sense because number is a JSON type and string is a JSON type. But I feel that somehow I dont have the big picture about that so I wanted to patch it first for the test since TextInputFilter expects a _deserialize function that takes `string | null | undefined` and it requires some refactoring

Reply:
I have added the comment about this